setup.py

```diff
@@ -23,11 +23,11 @@
     ],
     packages=find_packages(exclude=['tests']),
     install_requires=[
-        'django>=1.8,<1.9',
+        'django>=1.8,<2.0',
         'django-model-utils>=1.4.0',
         'djangorestframework>=3.2.0,<3.4.0',
         'djangorestframework-oauth>=1.1.0,<2.0.0',
-        'edx-django-oauth2-provider>=0.5.0,<1.0.0',
+        'edx-django-oauth2-provider>=1.2.0',
```

Comment:
Thanks @efischer19, that solved the problem!

![image](https://user-images.githubusercontent.com/484484/28084592-0c071686-6647-11e7-8917-479a39df0189.png)